src/common/components/ApprovedByField/ApprovedByField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { KeyValue } from '@folio/stripes/components';
+
+import { get } from 'lodash';
+
+import { USERS } from '../../resources';
+import { USERS_API } from '../../constants/api';
+
+const ApprovedByField = ({ approvedById, resources }) => {
+  const user = get(resources, ['user', 'records', 0]);
+  const approvedByName = `${get(user, 'personal.firstName', '')} ${get(user, 'personal.lastName', '')}`;
+
+  return (
+    <KeyValue
+      label={<FormattedMessage id="ui-invoice.invoice.details.information.approvedBy" />}
+      value={approvedById ? approvedByName : ''}
+    />
+  );
+};
+
+ApprovedByField.manifest = Object.freeze({
+  user: {
+    ...USERS,
+    GET: {
+      path: (queryParams, pathComponents, resourceValues, logger, props) => {
+        const approvedBy = props.approvedById;
+
+        if (approvedBy) {
+          return `${USERS_API}/${approvedBy}`;
```

Comment:
I think you can change the whole resource to this:
```
  approvedByUser: {
    ...BASE_RESOURCE,
    path: 'users/!{approvedByUserId}',
  },
```

Reply:
Fixed